tests/VolleyManagement.UnitTests/Services/ContributorService/ContributorTeamServiceTests.cs

```diff
@@ -45,16 +47,16 @@ public void GetAll_ContributorsTeamExist_ContributorsTeamReturned()
             MockRepositoryFindAll(testData);
 
             var sut = BuildSUT();
-            var expected = new ContributorTeamServiceTestFixture()
-                                            .TestContributors()
-                                            .Build()
-                                            .ToList();
+            ICollection<ContributorTeam> expected = new ContributorTeamServiceTestFixture()
+                .TestContributors()
+                .Build();
+                                            
 
             // Act
             var actual = sut.Get();
 
             // Assert
-            CollectionAssert.AreEqual(expected, actual, new ContributorTeamComparer());
+            CollectionAssert.AreEqual(expected as ICollection, actual as ICollection, new ContributorTeamComparer());
```

Comment:
Do not perfrom casts in assertions. If it fails error message will be misleading

Reply:
![image](https://user-images.githubusercontent.com/17709640/36486196-9ca176d8-1726-11e8-820e-dbc418ba6603.png)

Reply:
Look for other solutions